whitepaper/gridcoin-whitepaper-draft.md

```diff
@@ -662,8 +662,8 @@ Reddit
 Discord
 <https://discord.gg/jf9XX4a>
 
-Slack @todo: this link needs to be fixed
-<https://join.slack.com/t/teamgridcoin/shared_invite/enQtMjk2NTI4MzAwMzg0LTUzMmY0YjdiNzYxYzQ0MDc3MGE1NjQ3Nzg2NWMzZTUzMjAwZjdhMWI1YWUzMDE4YzQyZjVjMjMzOTc1MRmMmM>
+Slack
+<https://join.slack.com/t/teamgridcoin/shared_invite/enQtMjk2NTI4MzAwMzg0LTE4N2I3ZWZjYWJlZGM1Zjg3MTUyMDhiN2M5NmRmZTA2NDA0ZmY1ZTFmOGM3ZGU2YTBkOTdhNTk2ZjkzMGZkODY>
 
```

Comment:
Unrelated to this PR but can we show a bit love to Matrix(https://matrix.to/#/#gridcoin-community:matrix.org) and IRC(#gridcoin on irc.libera.chat)? A cryptocurrency which only lists proprietary messaging channels is a bit hypocritical.